src/thunks/updateUser.js

```diff
@@ -2,16 +2,21 @@ import { fetchData } from '../utils';
 import { setLoading, setError, setUser } from '../actions';
 import * as gql from '../queries';
 
-export const updateUser = (userInfo) => {
+export const updateUser = (userInfo, updateImage = false) => {
```

Comment:
Nice work updating this to take into account if we are updating just the user's image or all information for when they are updating their profile!